### PR TITLE
feat: debug screen

### LINKED ISSUE #113: 🚀 Feature: A3 debug (29)
- [ ] debug - slide 29

https://www.figma.com/file/7DzNiScbp6t3x0p86pz5P9/Playnets?node-id=1918%3A79119

blocks completion of #115 

---

* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by reginawang99**
closes #113 
- added hasSound to carousel

### CODE REVIEW

File: src/components/Activities/Activity3/Game/index.tsx [hunk 1]
```diff
@@ -91,7 +97,7 @@ function Game(): JSX.Element {
     [A3_GAME_STATE.PriorityWeighing]:
       <FeaturesSlidebar initialFeatureWeights={featureWeights} setFeatureWeights={setFeatureWeights} />,
     [A3_GAME_STATE.TimeAllocation]: <>3<DemoNextButton /></>,
-    [A3_GAME_STATE.DebuggingResults]: <>4<DemoNextButton /></>,
+    [A3_GAME_STATE.DebuggingResults]: <><DebuggingResults/><DemoNextButton /></>,
```
Comment (tr89on) [inline src/components/Activities/Activity3/Game/index.tsx:100]: v minor
```suggestion
    [A3_GAME_STATE.DebuggingResults]: <><DebuggingResults/></>,
```

File: src/components/shared/Carousel.tsx [hunk 2]
```diff
@@ -39,6 +39,7 @@ interface CarouselProps {
   subtitle?: string;
   onNext?: () => void;
   onPrev?: () => void;
+  hasSound: boolean;
```
Comment (tr89on) [inline src/components/shared/Carousel.tsx:42]: Is act 3 not supposed to have sound or is this like a temporary var?

**Review by tr89on — COMMENTED** [review]
seems aight

File: src/components/shared/Carousel.tsx (hunk 2 above)
Comment (reginawang99) [reply src/components/shared/Carousel.tsx:42]: nope! no sound

**Review by tr89on — APPROVED** [review]

**Review by tr89on — APPROVED** [review]

**Pull request merged**